Package.swift

```diff
@@ -43,6 +43,12 @@ let package = Package(
                 "SwiftyTextTable",
             ]
         ),
+        .testTarget(
+          name: "swiftlintTests",
```

Comment:
Can this be renamed/moved to `CLITests`?

Reply:
If you could also mirror this in Bazel here that'd be appreciated: https://github.com/realm/SwiftLint/blob/main/Tests/BUILD

If you're not comfortable with Bazel don't worry about it, I can do this as a follow-up.

Reply:
Yes, I'm ok with this

Reply:
Sorry, I replied to first comment and didn't see the second about Basel.
I have no experience with Basel and would be grateful if you take this (: